lib/extras.php

```diff
@@ -1964,7 +1964,7 @@ function novablocks_get_card_item_description( $description, $attributes ) {
 	ob_start();
 
 	if ( ! empty( $description ) && ! empty( $attributes['showDescription'] ) ) { ?>
-		<div class="novablocks-grid__item-content novablocks-card__description">
+		<div class="novablocks-grid__item-content novablocks-card__content">
```

Comment:
Not sure if it's recommended, but maybe `showDescription` attribute should be rename in `showContent` to match the markup.